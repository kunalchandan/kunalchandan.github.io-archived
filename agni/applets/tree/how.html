<!DOCTYPE HTML>

<html>
	<head>
		<!-- Global site tag (gtag.js) - Google Analytics -->
		<script async src="https://www.googletagmanager.com/gtag/js?id=UA-109129979-1"></script>
		<script>
		  window.dataLayer = window.dataLayer || [];
		  function gtag(){dataLayer.push(arguments);}
		  gtag('js', new Date());

		  gtag('config', 'UA-109129979-1');
		</script>
    <!-- Google Tag Manager -->
    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
    })(window,document,'script','dataLayer','GTM-M33JZZG');</script>
    <!-- End Google Tag Manager -->

		<title>Tree Explained</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="../../assets/css/main.css" />
	</head>
	<body class="subpage">
    <!-- Google Tag Manager -->
    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
    })(window,document,'script','dataLayer','GTM-M33JZZG');</script>
    <!-- End Google Tag Manager -->

		<!-- Header -->
			<header id="header">
				<div class="logo"><a href="index.html">Tree <span>by Kunal</span></a></div>
				<a href="../../menu">Menu</a>
			</header>

		<!-- Nav -->
			<nav id="menu">
				<ul class="links">
					<li><a href="../../index.html">Home</a></li>
					<li><a href="../../generic.html">Generic</a></li>
					<li><a href="../../elements.html">Elements</a></li>
				</ul>
			</nav>

		<!-- One -->
			<section id="One" class="wrapper style3">
				<div class="inner">
					<header class="align-center">
						<p>Fractal Tree</p>
						<h2>Behind the Scenes of the Fractal tree</h2>
					</header>
				</div>
			</section>

		<!-- Two -->
			<section id="two" class="wrapper style2">
				<div class="inner">
					<div class="box">
						<div class="content">
							<header class="align-center">
								<p>DANK CODE</p>
								<h2>Fractal Recursive Tree</h2>
							</header>
							<p>This is the Javascript that makes up the Fractal tree using p5.js </br>
							There are problems with indenting and HTML, so please copy and paste this code into any IDE and use the auto indent function.</p>

							<code>var length = 200;</br></code>
							<code>var PI = 3.14159265358979323846;</br></code>
							<code>var TWO_PI = PI*2;</br></code>
							<code></br></code>
							<code>var angle = 0;</br></code>
							<code>var slider;</br></code>
							<code></br></code>
							<code>function setup() {</br></code>
							<code>createCanvas(400, 400);</br></code>
							<code>slider = createSlider(0, TWO_PI, PI / 4, 0.01);</br></code>
							<code>}</br></code>
							<code></br></code>
							<code>function draw() {</br></code>
							<code>background(51);</br></code>
							<code>angle = slider.value();</br></code>
							<code>stroke(255);</br></code>
							<code>translate(200, height);</br></code>
							<code>branch(100);</br></code>
							<code></br></code>
							<code>}</br></code>
							<code></br></code>
							<code>function branch(len) {</br></code>
							<code>line(0, 0, 0, -len);</br></code>
							<code>translate(0, -len);</br></code>
							<code>if (len > 4) {</br></code>
							<code>push();</br></code>
							<code>rotate(angle);</br></code>
							<code>branch(len * 0.67);</br></code>
							<code>pop();</br></code>
							<code></br></code>
							<code>push();</br></code>
							<code>rotate(-angle);</br></code>
							<code>branch(len * 0.67);</br></code>
							<code>pop();</br></code>
							<code>}</br></code>
							<code>}</br></code>
						</div>
					</div>
				</div>
			</section>

		<!-- Footer -->
			<footer id="footer">
				<div class="copyright">
					&copy; Kunal The Chandan. All rights reserved.
				</div>
			</footer>

		<!-- Scripts -->
			<script src="../../assets/js/jquery.min.js"></script>
			<script src="../../assets/js/jquery.scrollex.min.js"></script>
			<script src="../../assets/js/skel.min.js"></script>
			<script src="../../assets/js/util.js"></script>
			<script src="../../assets/js/main.js"></script>

	</body>
</html>