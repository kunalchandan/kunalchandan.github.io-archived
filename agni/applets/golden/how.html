<!DOCTYPE HTML>

<html>
	<head>
		<!-- Global site tag (gtag.js) - Google Analytics -->
		<script async src="https://www.googletagmanager.com/gtag/js?id=UA-109129979-1"></script>
		<script>
		  window.dataLayer = window.dataLayer || [];
		  function gtag(){dataLayer.push(arguments);}
		  gtag('js', new Date());

		  gtag('config', 'UA-109129979-1');
		</script>

		<title>Square Explained</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="../../assets/css/main.css" />
	</head>
	<body class="subpage">
    <!-- Google Tag Manager -->
    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
    })(window,document,'script','dataLayer','GTM-M33JZZG');</script>
    <!-- End Google Tag Manager -->

		<!-- Header -->
			<header id="header">
				<div class="logo"><a href="index.html">Square <span>by Kunal</span></a></div>
				<a href="../../menu">Menu</a>
			</header>

		<!-- Nav -->
			<nav id="menu">
				<ul class="links">
					<li><a href="../../index.html">Home</a></li>
					<li><a href="../../generic.html">Generic</a></li>
					<li><a href="../../elements.html">Elements</a></li>
				</ul>
			</nav>

		<!-- One -->
			<section id="One" class="wrapper style3">
				<div class="inner">
					<header class="align-center">
						<p>Recursive Square</p>
						<h2>Behind the Scenes of the golden square</h2>
					</header>
				</div>
			</section>

		<!-- Two -->
			<section id="two" class="wrapper style2">
				<div class="inner">
					<div class="box">
						<div class="content">
							<header class="align-center">
								<p>How?</p>
								<h2>Recursive Square</h2>
							</header>
							<p>So this is the How for the Golden square thing, I made this in a very similar fasion to the recursive tree generator.</p>
							<a href="../tree/how.html" class="button special">How Tree?</a>
							<p></p>
							<p>This was also made in inspiration from Gold Plated Goof's video on his favourite math things.</p>
						</div>
					</div>
				</div>
			</section>

		<!-- Footer -->
			<footer id="footer">
				<div class="copyright">
					&copy; Kunal The Chandan. All rights reserved.
				</div>
			</footer>

		<!-- Scripts -->
			<script src="../../assets/js/jquery.min.js"></script>
			<script src="../../assets/js/jquery.scrollex.min.js"></script>
			<script src="../../assets/js/skel.min.js"></script>
			<script src="../../assets/js/util.js"></script>
			<script src="../../assets/js/main.js"></script>

	</body>
</html>
